.travis.yml

```diff
@@ -0,0 +1,5 @@
+script:
```

Comment:
Nice, this is a great minimal travis file. If you wanted to take this a bit further you could think about following the practices suggested in the [Travis docker docs](https://docs.travis-ci.com/user/docker/) e.g. explicitly specifying docker as a service, and logging in beforehand, but neither is important here